src/backends/torch/torchinputconns.cc

```diff
@@ -642,6 +677,55 @@ namespace dd
     return imgt;
   }
 
+  at::Tensor ImgTorchInputFileConn::target_to_tensor(const int &target)
+  {
+    at::Tensor targett{ torch::full(1, target, torch::kLong) };
+    return targett;
+  }
+
+  at::Tensor
+  ImgTorchInputFileConn::target_to_tensor(std::vector<double> &target)
+  {
+    int64_t tsize = target.size();
+    at::Tensor targett = torch::zeros(tsize, torch::kFloat32);
+    int n = 0;
+    for (auto i : target) // XXX: from_blob does not seem to work, fills up
```

Comment:
from_blob needs a .clone() afterwards, default behavior is to only use pointer to data , clone copies into a new tensor, maybe this is your problem here ?

Reply:
Thanks, I tried, but same spurious data output with `clone()`.